src/protocol.rs

```diff
@@ -505,6 +510,10 @@ mod tests {
 
         // Keygen is done! Makre sure there are no more messages.
         assert!(inboxes_are_empty(&inboxes));
+        // And make sure all participants have successfully terminated.
+        assert!(keygen_quorum
+            .iter()
+            .all(|p| *p.status() == crate::keygen::participant::Status::TerminatedSuccessfully));
```

Comment:
This is nice! Can you add the equivalent check up for auxinfo on L481, too?

Reply:
Good catch!